data_registry/models.py

```diff
@@ -118,6 +118,10 @@ class Meta:
     source_url = TextField(blank=True, verbose_name="source URL",
                            help_text="The URL of the publication.")
 
+    last_reviewed = DateField(blank=True, null=True, verbose_name="last reviewed",
+                              help_text="The date on which the most recent data feedback report was made. "
+                                        "Only the year and month are important.")
```

Comment:
We don't always write the summary based on a report. Also, I anticipate that if we review an old summary, we might check that it's still correct, and then update the date.

Shouldn't this field be after `summary`, for better flow in the code?

You'll need to delete and regenerate the migration.

```suggestion
                              help_text="The date on which the quality summary was last confirmed to be correct. "
                                        "Only the year and month are published.")
```